src/sonic-yang-models/yang-models/sonic-voq-inband-interface.yang

```diff
@@ -0,0 +1,56 @@
+module sonic-voq-inband-interface {
+    namespace "http://github.com/Azure/sonic-voq-inband-interface";
+    prefix voq-inband-intf;
+    yang-version 1.1;    
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC BGP Internal Neighbor for voq chassis platforms";
+
+    revision 2022-10-06 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-voq-inband-interface {
+        container VOQ_INBAND_INTERFACE {
+            description "VOQ inband interface in VOQ Chassis";
+            list VOQ_INBAND_INTERFACE_LIST {
+                key "name";
+                leaf name {
+                    type string {
+                        pattern "Ethernet-IB[0-9]+";
+                    }
+                }
+                leaf inband_type {
+                    description "Type of inband interface";
+                    type string {
+                        pattern "port|Port";                        
```

Comment:
what's the difference between port and Port?

Reply:
Just the case